src/transaction.js

```diff
@@ -0,0 +1,47 @@
+import { pool } from './pool';
+import { parseParametersTransaction } from './parser';
+import { slowQueryWarning, debug } from './config';
+
+const resourceName = GetCurrentResourceName() || 'oxmysql';
```

Comment:
Could you transfer that to `config.js` and re-use it in both execute and transaction?